arm9/src/bag.c

```diff
@@ -0,0 +1,463 @@
+#include "global.h"
+#include "MI_memory.h"
+#include "bag.h"
+#include "itemtool.h"
+#include "heap.h"
+
+#pragma thumb on
+
+u32 Sav2_Bag_sizeof(void)
+{
+    return sizeof(struct Bag);
+}
+
+struct Bag * Sav2_Bag_new(u32 heap_id)
+{
+    struct Bag * ret = (struct Bag *)AllocFromHeap(heap_id, sizeof(struct Bag));
+    Sav2_Bag_init(ret);
+    return ret;
+}
+
+void Sav2_Bag_init(struct Bag * bag)
+{
+    MI_CpuClear16(bag, sizeof(struct Bag));
+}
+
+void Sav2_Bag_copy(const struct Bag * src, struct Bag * dest)
+{
+    MI_CpuCopy8(src, dest, sizeof(struct Bag));
+}
+
+u32 Bag_GetRegisteredItem(struct Bag * bag)
```

Comment:
inconsistent naming

Reply:
Elaborate please?

Reply:
some funcs are called Bag_ and some are Bag without the underscore